latexpdf/Dockerfile

```diff
@@ -10,6 +10,8 @@ RUN apt-get update \
       \
       latexmk \
       lmodern \
+      # https://github.com/sphinx-doc/sphinx/blob/7ecf6b8/sphinx/builders/latex/constants.py#L38-L60
```

Comment:
I think this comment is not needed because it does not point to the current implementation. And I doubt the comment itself is really needed (we've never added comments for other packages).

Reply:
done